Source/WebKit/UIProcess/API/APIHTTPCookieStore.h

```diff
@@ -60,7 +52,7 @@ namespace API {
 
 class HTTPCookieStore;
 
-class HTTPCookieStoreObserver : public CanMakeWeakPtr<HTTPCookieStoreObserver> {
+class HTTPCookieStoreObserver : public RefCountedAndCanMakeWeakPtr<HTTPCookieStoreObserver> {
```

Comment:
It is unfortunate to go through the pain to make this RefCounted but then don't leverage it at all to improve safety. This PR should include the following improvement:
```
void HTTPCookieStore::cookiesDidChange()
{
    for (auto& observer : m_observers)
        observer.cookiesDidChange(*this);
}
```
->
```
void HTTPCookieStore::cookiesDidChange()
{
    for (Ref observer : m_observers)
        observer->cookiesDidChange(*this);
}
```
in APIHttpCookieStore.cpp.

Reply:
Oh, right. Thanks. I really want the static analyzer in EWS.